oshdb/src/main/java/org/heigit/ohsome/oshdb/OSHDBBoundable.java

```diff
@@ -0,0 +1,52 @@
+package org.heigit.ohsome.oshdb;
+
+public interface OSHDBBoundable {
+  
```

Comment:
AFAICS, this empty line (and quite a few more in this file and perhaps the whole PR) contains two spaces. Empty lines should be fully empty. In general, there should not be any trailing whitespace at the end of any line.

![image](https://user-images.githubusercontent.com/1927298/115588197-c7900800-a2ce-11eb-93f8-91f9caf0d94e.png)

Reply:
apparently our checkstyle ruleset does not complain about not empty blank lines. Maybe you should add this to our rules, so that it becomes more visible.
This should also be fixed.

Reply:
yes, because strictly speaking the java rules don't specify this as a rule (while the same style guides recommend to trim trailing whitespace for other languages such as C#, see https://github.com/google/styleguide/issues/262). but yes, we should think about adding it to our own rules. I will open a ticket for that right now.

Reply:
Sounds good, thank you. As I said this makes non empty lines or trailing whitespaces more prominent to remove :+1: